pkgs/tools/networking/openssh/default.nix

```diff
@@ -17,9 +17,10 @@ let
     sha256 = "682b4a6880d224ee0b7447241b684330b731018585f1ba519f46660c10d63950";
   };
 
+  # **please** update this patch when you update to a new openssh release.
```

Comment:
unfortunately the only way I know of is to find a newer version in the Debian source repo ... which may not be available right away when there is an openssh release :-/